test/regressions/TestViewer.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import clsx from 'clsx';
 import { useFakeTimers } from 'sinon';
 import { withStyles } from '@material-ui/styles';
-import { createTheme } from '@material-ui/data-grid';
+import { createTheme } from '../../packages/grid/_modules_/grid/utils/utils';
```

Comment:
Interesting, we can't use `@material-ui/data-grid/utils/utils` because of the `_modules_` folder structure. This folder structure prevents us to split MIT and Commercially licensed code. So we need to move most of the modules away. Once we do how do XGrid and DataGrid share the utils? I guess we have to either keep the utils in or export them as `unstable_`. Anyway, it's a problem for our future self. Making the utils private is the right thing to do.

Reply:
Yes, removing the unnecessary exports is a first step to help us reorganize the codebase between the MIT and the Commercially licensed code.

And there is some code that is not Grid related and should not be exported and not be in a `grid` folder but probably in a `_shared_` directory that will be accessed by the grids but also the new X components.
That's the case for a lot of the utils.

Reply:
>  but probably in a _shared_ directory that will be accessed by the grids but also the new X components.

If it needs to be access cross component, then we need a similar package to `@material-ui/utils`. Maybe we should name them:

- `@mui/core-utils` (prev `@material-ui/utils`)
- `@mui/x-utils`